lib/catalog/update_order_item.rb

```diff
@@ -1,31 +1,28 @@
 module Catalog
   class UpdateOrderItem
-    def initialize(topic, task, order_item = nil)
-      @payload    = topic.payload
-      @message    = topic.message
+    CRHC_PREFIX = 'expose_to_cloud_redhat_com_'.freeze
+
+    def initialize(task, order_item = nil)
       @task       = task
       order_item ||= OrderItem.find_by!(:topology_task_ref => @task.id)
       @order_item = order_item
     end
 
     def process
-      Rails.logger.info("Processing service order topic message: #{@message} with payload: #{@payload}")
```

Comment:
The message and payload are already logged when each message is received from Kafka, thus no need to repeat here.